### ISSUE TITLE
Wouldn't it be nice to have a client a2pi*.s on the GS?

### ISSUE THREAD

**Issue by david-schmidt** [dschmenk/apple2pi#6]
So, there's a2pi.s and a2piplus.s.  I guess a GS version would be a2pigs.s.  Some basic questions... how do the .s files get assembled (I assume by Merlin), how do they get invoked at the client end (I assume there is some init code in startup), and the coolest thing would probably be to autodetect which machine it was on and pick the appropriate client.

**Comment by dschmenk** [comment]
> On Feb 18, 2015, at 8:06 AM, David Schmidt <notifications@github.com> wrote:
> 
> So, there's a2pi.s and a2piplus.s. I guess a GS version would be a2pigs.s.  
> 

Yep. That is the network name of my IIgs. 
> Some basic questions... how do the .s files get assembled (I assume by Merlin), how do they get invoked at the client end (I assume there is some init code in startup), and the coolest thing would probably be to autodetect which machine it was on and pick the appropriate client.
> 

The machine detection is already in the startup program. It only differentiates between the ][+ and later models. I just never wrote a IIgs client because it wasn't high on my priority list and I don't have the correct serial cable (and I secretly hate the IIgs )
> —
> Reply to this email directly or view it on GitHub.
> 
If anyone wants to write it, I'll add it the next version.

**Comment by david-schmidt** [comment]
Yep, I finally found all the bits on the project.  I'll add a little more discriminating detector and a pigs file.  After one year - it seems you have your first IIgs customer.

(and I secretly hate the IIgs )
Amen, brother Davex.  I'll send you a serial cable if you want one.  But that removes an excuse, so I understand if you don't...

**Comment by david-schmidt** [comment]
First pass implemented in pull request #7.  It will be interesting to consider the case where an Apple II Pi card (with 6551 chip) is inserted into a GS.  I'm not sure how we would autodetect the card vs. an SSC card vs. the native serial SCC serial port(s) - and correctly decide which to use.  We might have to let autodetection fail in that case and ask the user to choose which driver to use (6551 vs. SCC-based).
